Hyper/Meshes/Chunk.cs

```diff
@@ -83,12 +95,11 @@ public bool Build(Vector3 location, float deltaTime, float brushWeight, int radi
         var x = (int)location.X - Position.X;
         var y = (int)location.Y - Position.Y;
         var z = (int)location.Z - Position.Z;
-
-        if (x < 0 || y < 0 || z < 0
-            || x > Size - 1 || y > Size - 1 || z > Size - 1
-            || _voxels[x, y, z].Value >= 1f)
+        
+        if (x < -radius || y < -radius || z < -radius
```

Comment:
why was that changed?

Reply:
this is because we want to take into consideration a position that we want to mine that is outside of the chunk. this fixes the problem of mining/building near the edges, but is also a bit more computationally expensive